src/NServiceBus.Core/Hosting/Helpers/AssemblyScanningComponent.cs

```diff
@@ -47,7 +47,11 @@ public static AssemblyScanningComponent Initialize(Configuration configuration,
             {
                 settings.AddStartupDiagnosticsSection("AssemblyScanning", new
                 {
-                    Assemblies = scannableAssemblies.Assemblies.Select(a => a.FullName),
+                    Assemblies = scannableAssemblies.Assemblies.Select(a => new
```

Comment:
Note that this is not considered a breaking change since we explicitly say that the format can change:

>The structure and format of the data produced should not be considered stable or parsable. Nodes may be added, removed, or moved in minor and patch versions.

https://docs.particular.net/nservicebus/hosting/startup-diagnostics